### ISSUE TITLE
add this component to HACS?

### ISSUE THREAD

**Issue by jurgenweber** [exxamalte/home-assistant-custom-components-nsw-rural-fire-service-fire-danger#14]
so I don't have to manually manage your custom components. :)

https://hacs.xyz/

**Comment by exxamalte** [comment]
Transferring the issue here. The repository itself is alreadt HACS compatible, but still need to add the URL to the default repositories.

**Comment by jurgenweber** [comment]
oh, I am using the other repo. I had no idea you had deprecated it. I assume that error I found is fixed here?

**Comment by exxamalte** [comment]
Sorry, haven't officially deprecated the other repository yet, but should do that. The error you found was fixed in the latest version (0.5) in this new repository.

**Comment by jurgenweber** [comment]
Yeah, the annoying part is I don't seem to be able to add a custom repo with HACS.... the UI is broken. :\

**Comment by exxamalte** [comment]
PR to add this as a default repository opened (but currently blocked by another PR): https://github.com/hacs/default/pull/754

**Comment by exxamalte** [comment]
This custom component has just been added to the default repositories.
